common/types/ballot.go

```diff
@@ -0,0 +1,166 @@
+package types
+
+import (
+	"bytes"
+
+	"github.com/spacemeshos/go-spacemesh/log"
+	"github.com/spacemeshos/go-spacemesh/signing"
+)
+
+const (
+	// BallotIDSize in bytes.
+	// FIXME(dshulyak) why do we cast to hash32 when returning bytes?
+	BallotIDSize = Hash32Length
+)
+
+// BallotID is a 20-byte sha256 sum of the serialized ballot used to identify a Ballot.
+type BallotID Hash20
+
+// EmptyBallotID is a canonical empty BallotID.
+var EmptyBallotID = &BallotID{}
```

Comment:
why is it a pointer? can we define it as a value and get a pointer when we need a pointer?

Reply:
done. was copying `EmptyATXID`